internal/packages/packages.go

```diff
@@ -243,10 +244,17 @@ func (dsm *DataStreamManifest) GetPipelineNameOrDefault() string {
 // for this data stream.
 func (dsm *DataStreamManifest) IndexTemplateName(pkgName string) string {
```

Comment:
As this isn't a standard, straightforward logic, would you mind adding some unit tests?

Reply:
Sure will do

Reply:
Done